frontend/integration-tests/protractor.conf.ts

```diff
@@ -149,7 +149,7 @@ export const config = {
   skipSourceMapSupport: true,
   jasmineNodeOpts: {
     print: () => null,
-    defaultTimeoutInterval: 60000,
+    defaultTimeoutInterval: 1000000,
```

Comment:
Woah, this is a big timeout. This seems a little tooo big. Why is 1000 seconds (1 3/4 minutes) needed for a timeout?

Reply:
Due to network (~100kbs) issues, scripts are failing with async timeouts, So increased the default timeout interval to re-verify the scripts results.. This will be removed once network issues are resolved from my side, Is it fine, Do you want me to replace the old value